.changeset/gorgeous-frogs-buy.md

```diff
@@ -0,0 +1,7 @@
+---
+'@primer/react-brand': patch
+---
+
+Added `hasBorder` prop to `Pillar` for alternative presentation
+
+🔗 (See Storybook for an example)[https://primer.style/brand/storybook/?path=/story/components-pillar-features--with-border]
```

Comment:
Not sure if this line is copy/pasted as I keep seeing it come up, but the parens are the wrong way round here.

```suggestion
🔗 [See Storybook for an example](https://primer.style/brand/storybook/?path=/story/components-pillar-features--with-border)
```

Reply:
oof thanks, yeah it's a copy/paste issue.